src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -210,6 +210,44 @@ def default_api_version(self):
         ResourceType.MGMT_IOTHUB: '2020-03-01',
         ResourceType.MGMT_ARO: '2020-04-30'
     },
+    '2020-09-01-hybrid': {
+        ResourceType.MGMT_STORAGE: '2019-06-01',
+        ResourceType.MGMT_NETWORK: '2018-11-01',
+        ResourceType.MGMT_COMPUTE: SDKProfile('2020-06-01', {
+            'resource_skus': '2019-04-01',
+            'disks': '2019-07-01',
+            'disk_encryption_sets': '2019-07-01',
+            'disk_accesses': '2020-05-01',
+            'snapshots': '2019-07-01',
+            'galleries': '2019-12-01',
+            'gallery_images': '2019-12-01',
+            'gallery_image_versions': '2019-12-01',
+            'virtual_machine_scale_sets': '2020-06-01'
+        }),
+        ResourceType.MGMT_KEYVAULT: '2019-09-01',
+        ResourceType.MGMT_RESOURCE_LINKS: '2018-05-01',
```

Comment:
Currently 2018-05-01 is not supported in python SDK. Please double check the api version.
See https://github.com/Azure/azure-rest-api-specs/tree/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01.
only 2016-09-01 has link support. https://github.com/Azure/azure-rest-api-specs/tree/master/specification/resources/resource-manager/Microsoft.Resources/stable/2016-09-01